examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i

```diff
@@ -36,7 +36,7 @@
 # Reference for graphite, stainless steel: Cincotti et al, DOI 10.1002/aic.11102
 # Assorted references for yttria, listed as comments in input file
 
-initial_temperature=873 #roughly 600C where the pyrometer kicks in
+initial_temperature = 873 #roughly 600C where the pyrometer kicks in
```

Comment:
If we're no longer using this variable in the input, do we want to keep it around at all?

Reply:
this variable still gets used as IC for blocks other than the stainless steel